influxdb/src/test/java/docs/javadsl/Cpu.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.javadsl;
+
+import java.time.Instant;
+import org.influxdb.annotation.Column;
+
+public class Cpu {
+
+  @Column(name = "time")
+  private Instant time;
+
+  @Column(name = "host", tag = true)
```

Comment:
Should the `name` be `"hostname"` here, to match the name of the variable it is decorating?